edit_profile/models.py

```diff
@@ -0,0 +1,34 @@
+from django.db import models
+from django.contrib.auth.models import User
+from django.utils.translation import gettext as _
+from django.templatetags.static import static
+
+
+class Profile(models.Model):
+    GENDER_MALE = 1
+    GENDER_FEMALE = 2
```

Comment:
The values for gender selection (GENDER_CHOICES) are labeled as “BTC” and “ETH”. This is an error. The correct values are “Male” and “Female”.